### DISCUSSION TITLE
DBreeze vs FASTER

### DISCUSSION THREAD

**Discussion by tiksn** [hhblaze/DBreeze#74] [Q&A] [answered]
DBreeze in many sense is similar to [FASTER](https://github.com/microsoft/FASTER)

What are the main differences?

For which usecase one should choose DBreeze or FASTER?

**Comment by hhblaze** [comment, marked as answer]
Hi, Tigran, we have released first version in 2012, that question was not a point at that time. Currently, DBreeze is very mature and stable, proposing a huge set of different database services integrated into C# language directly, working on many servers and mobile platforms.
Probably, Faster DB is just a competitor  covering a set of several DBreeze segments. 
This is not a point to discuss CONS and PROS unless we don't earn money on it.
If you are interested in such investigation, please, provide it.
For full set of DBreeze abilities - there is a nice documentation.
Slava Ukraine!!!

**Reply by tiksn** [reply]
Hi @hhblaze , Thanks for the answer.
Slava Ukraine!!!
